changelog.md

```diff
@@ -0,0 +1,59 @@
+# changelog
+
+## 1.0.0-rc1
+
+This is the first release targeting Qubes 4.2 and will require
+a full reinstall. As we have reached 1.0.0, we are transitioning
+from a closed pilot to an open beta program. If you are interested
+in using SecureDrop Workstation, please reach out to us at TBD.
```

Comment:
"please reach out to us via the support portal"?

Reply:
I like that, amending